public/index.css

```
@@ -4,6 +4,40 @@ body {
+  font-size: 0.82rem;
+}
+
+.generated-code-panel pre {
+  max-height: 320px;
+  overflow: auto;
+  padding: 12px;
+  color: #e6edf3;
+  background: #0d1117;
+  border-radius: 4px;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)